lua/cmake-tools/quickfix.lua

```diff
@@ -73,6 +73,10 @@ function _quickfix.run(cmd, env_script, env, args, cwd, opts, on_exit, on_output
           _quickfix.close(opts)
         end, 100)
       end
+      if code == 0 and not opts.auto_close_when_success then
+        _quickfix.show(opts)
+        _quickfix.scroll_to_bottom()
```

Comment:
When code equals 0, and `auto_close_when_success` is set to false, it should remain open. I think `_quickfix.show(opts)` is redundant, only `_quickfix.scroll_to_bottom()` is neccessary.

Reply:
I can reproduce this with this config. But the pr cannot fix it. I think the reason is from `vim.defer_fn`.

Reply:
I think we can remove this.